src/screens/AdminPortal/ManageTag/editModal/EditUserTagModal.tsx

```
@@ -1,47 +1,26 @@
+ * EditUserTagModal
+ *
+ * A modal form that allows administrators to edit existing user tags.
+ *
+ * @param editUserTagModalIsOpen - Boolean to control modal visibility
+ * @param hideEditUserTagModal - Function to close the modal
+ * @param newTagName - The current value of the tag name being edited
+ * @param handleEditUserTag - Function to save the changes
+ * @param setNewTagName - Function to update the tag name state
+ * @param t - Translation function
+ * @param tCommon - Common translation function
+ * @returns {JSX.Element} The rendered modal component
```

ESLint

[error] tsdoc-malformed-inline-tag: Expecting a TSDoc tag starting with "{@"

(tsdoc/syntax)